ui-v2/app/styles/components/table.scss

```diff
@@ -36,3 +36,36 @@ td dt.critical,
 td dt.critical + dd {
   color: $ui-color-failure;
 }
+/* Header Tooltips/Icon*/
+th {
+  overflow: visible;
+}
+th span {
+  @extend %tooltip;
+  @extend %with-info;
+  margin-left: 12px;
+  top: 3px;
+  width: 23px;
+  height: 15px;
+}
+th span:after {
+  left: -8px;
+}
+th span em::after {
+  @extend %tooltip-tail;
+  @extend %blink-in-fade-out;
+  top: auto !important;
```

Comment:
More of a question but why is `!important` required here? Is this the procedure for overrides when utilizing `@extend`?

Reply:
It's not so much procedure, but sometimes a consequence of things needed that you didn't plan for - it's not ideal!

`@extend` is basically a tool to help you make abstract and/or reusable components. As always with these things you often find yourself in a position where something that uses the reusable component can reuse all of it _apart_ from one thing. _Generally_ CSS selector specificity will help there, but occasionally it can't help, the plan didn't take everything into account - in which case `!important` is the sledge hammer to just say 'no, I really really want to overwrite that value'.

The alternative would be to go back to the reusable component and alter the 'plan' so you don't need `!important`. But that might effect other things without you realising (CSS regression testing can help here), so I always think it's best to do that as a whole looking at all the components that reuse this component, rather than just the one. So basically during a refactor PR rather than a feature PR.

In this PR I didn't change the `%tooltip-tail` component at all, so no risk of breaking it in other places. But the good thing is I think I'm using this component now in about 3 different ways/places, so it's ripe for a refactor if I want to go there.

I've re-planned and removed some other `!important`s in some of my other CSS refactor PR's I think.

All in all `!important` usage is contentious, so I generally think 3 times before using it. But, it's not specifically related to `@extend`.